config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  root 'users#index' 
+  resources :users, only: [:index, :show] do
+     resources :posts, only: [:show]
+  end
+  get 'posts', to: 'posts#index'
```

Comment:
- [ ] Kindly update your route to be like this:
```ruby
 resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```
as of now the access to the `post#index` page will be with route `/posts` instead of `/users/:user_id/posts` which was requested in this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project_v1_1.md#:~:text=Make%20sure%20that%20when%20you%20run%20the%20rails%20routes%20command%20in%20your%20console).

Reply:
Hi @gemmen29 

I hope you are doing well can we have a quick call I think is some confusion there 
as far I conclude 
- [ ] I need only three route 
     - user=> index, show
     - post => show
and only these routes I make all the wireframe requirement checked  as per my knowledge and for routing I added the extra index because I saw in the route requirement image (I don't need any action for or page for index for displaying posts by specific user (please check this commit => [commit](https://github.com/PowerLevel9000/Blog/tree/f21aa6f7182ca16982ccc673610973d65ff3a6bf) )

but as the Previous CR (@TedLivist ) suggested I made a new Post feed for the index where I can see all the posts made by different users in current changes 

Kindly,  guide me 😟😟 what should I do 

Curious Programer 
@PowerLevel9000

Reply:
![image](https://github.com/PowerLevel9000/Blog/assets/116451144/0feb3da7-54a9-4911-8a4d-474a1993632f)

Reply:
Hi @PowerLevel9000 you need four routes not three. You can see that in the wireframe and routes in the requirement. For the post controller, you need to provide the index route as in this wireframe
![blog_user_all_posts](https://github.com/PowerLevel9000/Blog/assets/24703171/6bdb7860-6c8b-421a-9d7a-593fa5b5f390). You did all the requirements of the project you just need to update your route, so you can access it with URL `/users/:user_id/posts` instead of creating an entirely new route that you can access with `/posts`. so just update your route to be:
```ruby
 resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```
This will create the missing route in the wireframe and requested routes in the requirement. Which is the `post#index` page.

Reply:
Ohh I Got this so there are two pages where we need to show Posts 

- [ ]  There are two pages where we need to show comments one for the dashboard(here we need to use recent five comments I think) and one for seeing all posts thanks for Clearing this  I really appreciate 🙌🙌🙌💕💕